### PR TITLE
fix(docs): fix `withTheme` example, add kkemple to contributors

### PR DESCRIPTION
**Pull request by kkemple**
**What**:
Fixed an error in the docs for `withTheme` example. I fixed in code sandbox example and wanted to reflect the correct example here as well. Also added myself to contribs since I forgot to add to `withTheme` PR.

**Why**:
If a user copied the example for `withTheme` it would not work properly.

**How**:
Updated README

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -150,6 +150,16 @@
...
       ]
+    },
+    {
+      "login": "kkemple",
+      "name": "Kurtis Kemple",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/3629876?v=3",
+      "profile": "https://twitter.com/kurtiskemple",
+      "contributions": [
+        "code",
+        "doc"
```
Comment (kentcdodds) [inline .all-contributorsrc]: Want to add test here too? See the contributing guidelines for how to regenerate after you update this 😉 if your want to.

Comment (kkemple) [inline .all-contributorsrc]: oh, i missed test, yeah, i'll update!

**Comment by kkemple** [comment]
thanks! 🎉

**Comment by kentcdodds** [comment]
Thank YOU!

**Pull request merged**